.vscode/settings.json

```diff
@@ -1,22 +1,21 @@
 {
-	"editor.codeActionsOnSave": {
-		"source.fixAll.eslint": "explicit"
-	},
-	"[typescript]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint"
-	},
-	"eslint.format.enable": true,
-	"files.exclude": {
-		"node_modules": true,
```

Comment:
this confused me for a while, I don't see why it's useful to hide `node_modules`